config/wdio/base.conf.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs-extra')
+const path = require('path')
+
+const REPORT_OUTPUT_DIR = path.resolve(__dirname, '../../reports/functional-tests')
```

Comment:
Maybe we can get this path from `.env` file or same, it's possible?

Reply:
I could not find any case for having reports directory other than predefined. Do you know such?